app/views/static/stgall/21198-zz00161mpp-msdesc-updated.html.erb

```diff
@@ -87,9 +87,16 @@
                                                 <p>
                                                     Stewart-Sykes, Alistair. <i>On the Lord's Prayer</i>. New York: St Vladimir's Seminary Press, 2004.
                                                 </p>
-                                                <p>
-                                                        <a target="_new" href="http://cyprianproject.info/DeDominicaOratione.htm">The Cyprian Project</a> <a target="_new" href="http://apps.brepolis.net/bmb/search.cfm?action=search_advanced_result&amp;startrow=1&amp;allindexterms=%22Cyprian, bishop of Carthage%22">IMB</a> <a target="_new" href="http://apps.brepolis.net/Lexiema/test/LinkingIn.aspx?id=e638450">LexMA</a>. 
-                                                </p>
+                                                <ul class="bibliography-links">
```

Comment:
@pghorpade can we refactor the html when there are imb / lexma links in the bibliography to be an unordered list with the `bibliography-links` class?